### PR TITLE
🌐 Add Spanish translation for `docs/es/docs/tutorial/body-nested-models.md`

### PR DESCRIPTION
**Pull request by chesstrian**

### CODE REVIEW

File: docs/es/mkdocs.yml [hunk 1]
```diff
@@ -37,7 +37,7 @@ plugins:
     data: data
 nav:
 - FastAPI: index.md
-- Languages:
+- Idiomas:
```
Comment (tiangolo) [inline docs/es/mkdocs.yml:40]:
```suggestion
- Languages:
```

This one has to be kept in English.

**Review by LaurEars — APPROVED** [review]
Looks good to me - includes changes requested by @tiangolo

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1faf27c71fb78382411906f4880e2204fe1c834b at: https://64922163cd6d1b0a552b6bdc--fastapi.netlify.app

**Comment by tiangolo** [comment]
📝 Docs preview for commit 1faf27c71fb78382411906f4880e2204fe1c834b at: https://649221647c02450ae0ca5bf7--fastapi.netlify.app

**Comment by codespearhead** [comment]
Do you happen to know two other Spanish-speaking developers that you can ask a [PR review](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/requesting-a-pull-request-review) from? [This](https://github.com/tiangolo/fastapi/pulls?q=is%3Apr+Spanish) might help.

**Comment by tiangolo** [comment]
This was covered in another PR, so I'll close this one now, thanks for the effort! :cake: :coffee:

**Pull request closed without merging**